components/generic_modal.scss

```diff
@@ -216,6 +216,32 @@
                     }
                 }
 
+                &.delete {
+                    @include primary-button;
+                    @include button-medium;
+
+                    border-color: transparent;
+                    background: $delete-color;
```

Comment:
instead of `$delete-color`, should be `--dnd-indicator`

Reply:
Done 👍